packages/core/test/lib/base.test.ts

```diff
@@ -1097,19 +1097,7 @@ describe('BaseClient', () => {
       const client = new TestClient(options);
 
       const transaction: Event = {
-        contexts: {
-          trace: {
-            op: 'pageload',
-            span_id: 'a3df84a60c2e4e76',
-            trace_id: '86f39e84263a4de99c326acab3bfe3bd',
-          },
-        },
-        environment: 'production',
-        event_id: '972f45b826a248bba98e990878a177e1',
-        spans: [],
-        start_timestamp: 1591603196.614865,
-        timestamp: 1591603196.728485,
```

Comment:
So this is just stuff we didn't need for the teest?

Reply:
Yup. It has nothing to do with the name-change data, so in the end it's just noise.